### ISSUE TITLE
keyboard view during playback

### ISSUE THREAD

**Issue by sugizo** [powertab/powertabeditor#457]
**Description**
nice to have keyboard view during playback

**Additional context**
tux guitar and musescore already have this feature

![image](https://github.com/powertab/powertabeditor/assets/2418295/7f13a956-646f-4381-a04a-ae94fc58a9e1)

thanks and best regards

**Comment by NalihznehC** [comment]
Good idea! Will be happy if we have some simple visualisations that do not take up much space.

**Comment by sugizo** [comment]
not sure, if your request : "do not take up much space"
because some softwares that support keyboard view use 88 keys piano
their option is just enable or disable keyboard view, 
some just readjust the height or width of keyboard

**on musescore**
![image](https://github.com/powertab/powertabeditor/assets/2418295/e4b133b6-36ad-4032-b086-5ba5b1a4ac56)
